packages/components/src/components/common/tabs/tabs.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { Tabs, TabsContent, TabsList, TabsTrigger } from './tabs';
+import { IvyIcon } from '../icon/icon';
+import { IvyIcons } from '@axonivy/ui-icons';
+import { Flex } from '@/components';
+
+const meta: Meta<typeof Tabs> = {
+  title: 'Common/Tabs',
+  component: Tabs,
+  argTypes: {
+    orientation: { control: 'select', defaultValue: 'horizontal', options: ['horizontal', 'vertical'] }
+  }
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Tabs>;
+
+export const Default: Story = {
+  render: props => (
+    <Tabs defaultValue='attribute' className='w-[400px]' {...props}>
+      <Flex direction={props.orientation === 'horizontal' ? 'column' : 'row'} gap={4}>
```

Comment:
Why is this flex needed? If I look here this is not needed: https://www.radix-ui.com/primitives/docs/components/tabs

Reply:
to put some space between the tabs and the content :D
Only to make the story looks a little bit better. can also use plain CSS like the radix-ui doc does it ;)